R/bucket_list.R

```diff
@@ -73,7 +77,8 @@ bucket_list <- function(
   orientation = c("horizontal", "vertical")
 ) {
 
-  # assert_that(is_header(header))
+  assert_that(is_header(header))
+  if (is.na(header)) header <- NULL
```

Comment:
Protect against `is.na(NULL) # logical(0)`

```suggestion
  if (isTRUE(is.na(header))) header <- NULL
```